tempesta_fw/t/functional/tempesta_test.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+from helpers import shell, remote
+
```

Comment:
1. Missing copyright. 
2. Need to document in Readme.md, why the file is needed, file docstring is required. Just to make clear that the file is not intended to be started manually